de.fabrik19.mos-Launcher.metainfo.xml

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+    <id>de.fabrik19.mos-Launcher</id>
+
+    <name>mos. Launcher</name>
+    <summary>Used for developing with the Mobilitysuite.</summary>
+
+    <metadata_license>CC0-1.0</metadata_license>
+    <project_license>LicenseRef-proprietary</project_license>
+
+    <description>
+        <p>
+            Used for developing with the Mobilitysuite.
```

Comment:
Description should be something more than summary.

Reply:
Added a better description.